test/lib/ansible_test/_data/requirements/constraints.txt

```
@@ -39,6 +39,7 @@ pyvmomi < 6.0.0 ; python_version < '2.7' # pyvmomi 6.0.0 and later require pytho
 pyone == 1.1.9 # newer versions do not pass current integration tests
 botocore >= 1.10.0 # adds support for the following AWS services: secretsmanager, fms, and acm-pca
 setuptools < 45 ; python_version <= '2.7' # setuptools 45 and later require python 3.5 or later
+virtualenv < 20.0.0 ; python_version <= '2.7'  # virtualenv 20.0.0 breaks on macOS due to the older pip we have there not installing the zipp dependency
```

Move this constraint below the other one for `virtualenv` and update as follows:

```suggestion
virtualenv < 20.0.0 ; python_version == '2.7'  # virtualenv 20.0.0 breaks on macOS due to the older pip we have there not installing the zipp dependency
```

---

Done.

---

I'm pretty sure it makes sense to have two constraints for `virtualenv` now. Let me know if that's incorrect.